### ISSUE TITLE
[enhanced-parts] VM implementation

### ISSUE THREAD

**Issue by eernstg** [dart-lang/sdk#56199]

**Comment by davidmorgan** [comment]
@natebiggs @alexmarkov Did anyone ever start looking at VM work for Parts With Imports (previously known as Enhanced Parts)?

**Comment by alexmarkov** [comment]
Not really. I think it should be mostly (if not fully) a front-end feature.

@johnniwinther Do you have a list of kernel AST changes which should be supported in the back-ends for this feature? Anything related to deferred loading, how we represent import/export etc?

**Comment by johnniwinther** [comment]
The CFE is currently feature complete and no changes have been made to the AST to accommodate this, it is even already used for patching.

That been said, we might want to test features, like deferred loading, that are not exercised in patching, explicitly, since we might have new patterns of encoding that backends do not anticipate.
